frontend/src/metabase/core/components/EditableText/EditableText.tsx

```diff
@@ -60,17 +69,36 @@ const EditableText = forwardRef(function EditableText(
     }
   }, [initialValue, previousInitialValue]);
 
+  useEffect(() => {
+    if (isMarkdown && isInFocus) {
+      input.current?.focus();
+    }
+  }, [isInFocus, isMarkdown]);
+
   const handleBlur = useCallback(
     e => {
+      if (isMarkdown) {
```

Comment:
nit: Maybe remove `isMarkdown` check and always set it to `false`? So `isInFocus` will be always correct